paramnb/__init__.py

```diff
@@ -184,7 +184,11 @@ def _make_widget(self, p_name):
         widget_class = wtype(p_obj)
 
         value = getattr(self.parameterized, p_name)
-        kw = dict(tooltip=p_obj.doc, value=value)
+
+        kw = dict(value=value)
+        if p_obj.doc:
```

Comment:
I briefly thought this should be ``if p_obj.doc is not None:``, based on honoring the title of the pull request, but doc==0 isn't a valid docstring and I guess we also want doc=="" to skip installing the tooltip, so I suppose it's better as it is.